src/components/Dialogs/elements/MultiOption.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <div>
+        <p class="mb-2 text-sm">
+            {{ modifiedItem === 'Task' ? 'Subtasks' : 'Board Columns' }}
+        </p>
+
+        <div class="flex flex-col gap-3">
+            <div
+                v-for="item in selectedMultiOptionItems"
+                class="flex items-center gap-2"
+            >
+                <input-text
+                    :placeholder="
+                        modifiedItem === 'Task' && formType === 'Add' ? item : undefined
+                    "
+                    :inputValue="
+                        modifiedItem === 'Task' && formType === 'Add' ? undefined : item
+                    "
```

Comment:
what?

Reply:
Ehm... so, this is the thing I was talking to you about today. I have the same form for editing and adding task/board and when editing I want to pre-render values, whereas when adding I want put this content into placeholders 🙃.